src/PatchKit.Api/ApiConnection.cs

```diff
@@ -209,27 +238,51 @@ private bool IsWithin(int value, int min, int max)
             return value >= min && value <= max;
         }
 
+        public IApiResponse Get(string path, string query)
+        {
+            return CreateResponse(() => {
+                Logger.LogDebug($"Getting response for GET request with path: '{path}' and query: '{query}'...");
+
+                return new Request
+                {
+                    Path = path,
+                    Query = query,
+                    MainServerExceptions = new List<Exception>(),
+                    CacheServersExceptions = new List<Exception>()
+                };
+            });
+        }
+
+        public IApiResponse Post(string path, string query, string data)
+        {
+            return CreateResponse(() => {
+                Logger.LogDebug($"Getting response for POST request with path: '{path}', query: '{query}' and data '{data}'...");
+
+                return new Request
+                {
+                    Path = path,
+                    Query = query,
+                    Method = "POST",
+                    Data = data,
+                    MainServerExceptions = new List<Exception>(),
+                    CacheServersExceptions = new List<Exception>()
+                };
+            });
+        }
+
         /// <summary>
         /// Retrieves specified resource from API.
         /// </summary>
         /// <param name="path">The path to the resource.</param>
         /// <param name="query">The query of the resource.</param>
         /// <returns>Response with resource result.</returns>
         /// <exception cref="ApiConnectionException">Could not connect to API.</exception>
-        public IApiResponse GetResponse(string path, string query)
+        private IApiResponse CreateResponse(Func<Request> requestBuilder)
```

Comment:
`GetResponse` was bad, but I think that `SendRequest` would be much more appropriate.